packages/bot-web-ui/src/components/dashboard/dashboard-component/load-bot-preview/delete-dialog.tsx

```diff
@@ -70,7 +66,7 @@ const DeleteDialog = ({
                     </Text>
                 </div>
                 <div>
-                    <Text color='prominent' line_height='s' size='xs'>
+                    <Text color='prominent' line_height='xxl' size='xs'>
```

Comment:
@farabi-deriv could you please check the desktop version design and all related things? I think for desktop should be another option size...

Reply:
@maryia-matskevich-deriv yes i checked. its 20px on figma so it should be xl. updated it.